build/configs/artik053/tc/defconfig

```diff
@@ -157,6 +158,11 @@ CONFIG_S5J_WDT_DEFTIMEOUT=7000
 CONFIG_S5J_SFLASH=y
 # CONFIG_S5J_SENSOR_PPD42NS is not set
 
+#
+# S5J PM Options
```

Comment:
1. I can't find this comment on Kconfig. I guess you modify this file manually.
At start of defconfig. below is written
```
# Automatically generated file; DO NOT EDIT.
```
Don't modify this file manually.

2. I don't think this is enough to show different title.

Reply:
Rectified defconfig changes